sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js

```diff
@@ -3257,12 +3258,16 @@ goog.scope(function() {
                 access = colorBuf.getSubregion(colorArea);
 
                 if (!maskUsed)
-                    access.clear(value);
+                    access.clear(isSRGB ? tcuTextureUtil.linearToSRGB(value) : value);
```

Comment:
Can you align the code with c++? c++ dealt with sRGB conversion, and it's simply lost in translation.